src/utils/currencies.ts

```diff
@@ -25,6 +25,8 @@ export const currencyForAnalytics: {
   [CiCoCurrency.cREAL]: 'cReal',
   [CiCoCurrency.ETH]: 'ETH',
 }
+export const symbolToAnalyticsCurrency = (symbol: string): string =>
+  symbol === 'CELO' ? 'cGLD' : symbol === 'cREAL' ? 'cReal' : symbol
```

Comment:
Needed to make backwards compatible with how analytics are being done currently.